docs/user-manuals/slo-config.md

```diff
@@ -0,0 +1,408 @@
+              "kubernetes.io/kernel": "anolis"
+            }
+          },
+          "policy": "cfsQuotaBurstOnly",
+          "cfsQuotaBurstPercent": 400
+        }
+      ]
+    }
+  # The configuration for system-level settings.
+  # Related features: SystemConfig.
+  # - clusterStrategy: the cluster-level configuration.
+  # - nodeStrategies: the node-level configurations which matches the nodes via the node selector and overrides the cluster configuration.
+  # - minFreeKbytesFactor: the factor for calculating the global minimum memory free watermark `/proc/sys/vm/min_free_kbytes`. `min_free_kbytes = minFreeKbytesFactor * nodeTotalMemory / 10000`.
+  # - watermarkScaleFactor: the reclaim factor `/proc/sys/vm/watermark_scale_factor` in once global memory reclaim.
+  # - memcgReapBackGround: whether to enable the reaper for orphan memory cgroups.
+  system-config: |-
+    {
+      "clusterStrategy": {
+        "minFreeKbytesFactor": 100,
+        "watermarkScaleFactor": 150,
+        "memcgReapBackGround": 0
+      }
+      "nodeStrategies": [
+        {
+          "name": "anolis",
+          "nodeSelector": {
+            "matchLabels": {
+              "kubernetes.io/kernel": "anolis"
+            }
+          },
+          "minFreeKbytesFactor": 100,
+          "watermarkScaleFactor": 150
+        }
+      ]
+    }
+```
+
+For more information, please check the user manuals and designs of the related features.
+
+## Quick Start
+
+1. Check the current SLO configurations via the ConfigMap `koordinator-system/slo-controller-config`.
+
+```bash
+$ kubectl get configmap -n koordinator-system slo-controller-config -o yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    meta.helm.sh/release-name: koordinator
+    meta.helm.sh/release-namespace: default
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: slo-controller-config
+  namespace: koordinator-system
+data:
+  colocation-config: |
+    {
+      "enable": false,
+      "metricAggregateDurationSeconds": 300,
+      "metricReportIntervalSeconds": 60,
+      "cpuReclaimThresholdPercent": 60,
+      "memoryReclaimThresholdPercent": 65,
+      "memoryCalculatePolicy": "usage",
+      "degradeTimeMinutes": 15,
+      "updateTimeThresholdSeconds": 300,
+      "resourceDiffThreshold": 0.1
+    }
+  resource-threshold-config: |
+    {
+      "clusterStrategy": {
+        "enable": false
+      }
+    }
+```
+
+2. Edit the ConfigMap `koordinator-system/slo-controller-config` to change the SLO config.
+
+```bash
+$ kubectl edit configmap -n koordinator-system slo-controller-config
+```
+
+For example, the configmap is edited as follows:
+
+```yaml
+data:
+  # ...
+  resource-threshold-config: |
+    {
+      "clusterStrategy": {
+        "enable": true,
+        "cpuSuppressThresholdPercent": 60,
+        "cpuSuppressPolicy": "cpuset",
+        "memoryEvictThresholdPercent": 60
+      }
+    }
+```
+
+3. Verify if the NodeSLO is successfully dispatched.
```

Comment:
default value will be omitted in NodeSLO

Reply:
added below